main.go

```diff
@@ -334,30 +333,31 @@ func mainCore() error {
 
 	blockDataSavers = append(blockDataSavers, explore)
 
-	// Sync up with the blockchain
+	// Coordinated sync up both sqlite and postgres DBs with the network. This
```

Comment:
Coordinated -> Coordinate. 
feels better.
"Coordinate the sync of both sqlite and postgres DBs with the network..."

If one day we decided to change the auxiliary db from Postgres to maybe Mysql or even a graph database, shall we be forced to rename the all Postgres occurrences to that specific db then. IMO I think we should use "auxiliary db" when referencing Postgres db now.